frontend_lib/src/helper.js

```diff
@@ -533,7 +533,7 @@ export const permissiveNumberEqual = (var1, var2) => {
 }
 
 export const createSpaceArborescence = spaceList => {
-  const spaceListWithChildren = spaceList.map(space => ({ ...space, children: [] }))
+  const spaceListWithChildren = spaceList.map(space => space.children ? space : { ...space, children: [] })
```

Comment:
Why?

Reply:
if a space already has children, we shouldn't replace the array for a empty one

Reply:
I updated to doesn't modify the original object.